src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -115,6 +116,7 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 _committedCounter ??= new PollingCounter("gc-committed", this, () => ((double)GC.GetGCMemoryInfo().TotalCommittedBytes / 1_000_000)) { DisplayName = "GC Committed Bytes", DisplayUnits = "MB" };
                 _exceptionCounter ??= new IncrementingPollingCounter("exception-count", this, () => Exception.GetExceptionCount()) { DisplayName = "Exception Count", DisplayRateTimeScale = new TimeSpan(0, 0, 1) };
                 _gcTimeCounter ??= new PollingCounter("time-in-gc", this, () => GC.GetLastGCPercentTimeInGC()) { DisplayName = "% Time in GC since last GC", DisplayUnits = "%" };
+                _totalGcTimeCounter ??= new IncrementingPollingCounter("total-time-in-gc", this, () => GC.GetTotalPauseDuration().TotalMilliseconds) { DisplayName = "Accumulated time spent in GC", DisplayUnits = "ms" };
```

Comment:
@reyang something to document.

Reply:
Curious why `ms` is used instead of `ns` or `s`?

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Runtime#processruntimedotnetgcduration

Reply:
> https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Runtime#processruntimedotnetgcduration

Happy to see this!

Reply:
> Curious why `ms` is used instead of `ns` or `s`?
> 

For EventCounters we've already got precedent in the `time-in-jit` counter below:  https://github.com/dotnet/runtime/pull/88699/files#diff-e5852984887004e4d639a743f3e005151014e27b5f09cc8ec313084d70191a6eL127

When we add a System.Runtime Meter in the future I'd match OpenTelemetry precedent.